package.json

```diff
@@ -5,21 +5,56 @@
   "private": true,
   "jsdelivr": "dist/index.umd.min.js",
   "workspaces": [
-    "utils/*",
-    "packages/*",
-    "components/*",
-    "demos/dashy",
-    "demos/dermatology",
-    "demos/gallery",
-    "demos/h3",
-    "demos/ie11",
-    "demos/imdb",
-    "demos/quickstart/*",
-    "demos/react/*",
-    "tests/*"
+    "xutils/*",
+    "xpackages/api",
```

Comment:
lots of references to "xpackages" in here. is that because you're mostly only concerned with comms?

Reply:
Its just my way of excluding the ones that have not been migrated yet...